src/main/resources/static/app/request/upload-file.html

```diff
@@ -83,7 +83,7 @@ <h4 class="h4 h4-palga">{{'Excerpt list'|translate}}</h4>
     </div>
 
     <div ng-if="isExcerptSelectionState(request.status)
-        && (request.paReportRequest || request.materialsRequest || request.clinicalDataRequest)
+        && (request.paReportRequest || Request.isMaterialsRequest(request) || request.clinicalDataRequest)
```

Comment:
`Request.isMaterialsRequest(request)` should be `isMaterialsRequest(request)` here